File: String/1108. Defanging an IP Address.go
package main

import "fmt"

func main() {
	fmt.Println(defangIPaddr("1.1.1.1"))
}

func defangIPaddr(address string) string {

	res := ""

	for i := 0; i < len(address); i++ {
		if address[i] == '.' {
			res += "[.]"
		} else {
			res += string(address[i])
		}
	}
	return res
}

File: LeetCode/LeetCode/String/1108. Defanging an IP Address.swift
//
//  1108. Defanging an IP Address.swift
//  LeetCode
//
//  Created by xiaoma on 2019/12/30.
//  Copyright © 2019 xiaoma. All rights reserved.
//

import Foundation


class Solution {
    func defangIPaddr(_ address: String) -> String {
        return address.replacingOccurrences(of: ".", with: "[.]")
    }
}
